.gitignore

```diff
@@ -18,6 +18,7 @@
 
 # Fortran module files
 *.mod
+!go.mod
```

Comment:
This rule seems redundant.

Reply:
I'm pretty sure we need that exception, so git doesn't ignore changes in that file.